### ISSUE TITLE
docs(listQuotes): Add author param 

### ISSUE THREAD

**Issue by raxraj** [lukePeavey/quotable#48]
In the `/quotes`, we have a query parameter to fetch quotes by using the author name Screenshot attached.
![image](https://user-images.githubusercontent.com/41513907/106910916-b0a63480-6727-11eb-8996-2c7e824e2887.png)

__But It is not documented__.
It is required to be updated.

**Comment by lukePeavey** [comment]
@raxraj 

The` name` param is going to be replaced by several other features in the near future. I removed it from the public docs pending these changes in case it ends up getting removed. 


### Search for authors by name

The `name` param uses regex matching to support search-like behavior. This feature is being replaced by a separate search endpoint (`/search/authors/`) that is intended to power a search bar to find authors by name. The new endpoint uses [Atlas Search](https://docs.atlas.mongodb.com/atlas-search/) instead of regex. It supports autocomplete and sorts the results by score.  This feature is complete, but is held up by something else. 

### Get a single author by name

It would still be useful to have a way to get a specific author by name. I am thinking about adding a `slug` field to authors, where the value would be a URL-safe version of the author name (ie `"john-f-kennedy-jr"`).  The `slug` could be used instead of an `id` to get a specific author or filter quotes by author. 

Possibly...

```HTTP
GET /authors/slug/:slug
```
### TL;DR

The `name` param will probably still be supported going forward to prevent breaking any existing projects. So if you think we should keep it in the docs until these other changes are made, I'd be fine with that. Feel free to submit a PR.

**Comment by raxraj** [comment]
Okay! @lukePeavey ,
Let me get that done, I think it must be in the doc, and whenever we get the slug feature back in, we can remove the same.

**Comment by lukePeavey** [comment]
Ok, sounds good.

**Comment by lukePeavey** [comment]
I forgot that quotes don't have an `authorId` field anymore.

Right now, the `name` param is only way to get additional details about the author of a quote returned by the `/random` or `/quotes` endpoint.  So you're right, it should definitely be in the docs until we add another way to do that.

**Comment by lukePeavey** [comment]
Sorry, I just realized that I misread your issue. I thought you were talking about the **`name`** param on the `/authors` endpoint 🤪 .

**Comment by lukePeavey** [comment]
Anyway, both params were removed from the docs at the same time pending possible changes. But I agree it makes sense to leave it in the docs in the meantime.

**Comment by raxraj** [comment]
It's an honest mistake :) Awesome.

**Comment by lukePeavey** [comment]
Thanks!
